lib/workers/repository/model/commit-message.ts

```diff
@@ -43,6 +45,14 @@ export class CommitMessage {
 
     return [prefix, message].join(' ').trim();
   }
+  public static formatCasing(commitMessage: string): string {
+    return commitMessage.replace(
+      regEx(/^([\w]+:|[\w]+\(.*\):)/).test(commitMessage)
+        ? regEx(/: \w+/)
+        : regEx(/^\w+/),
+      (match) => match.toLowerCase()
+    );
+  }
```

Comment:
wrong merge?

Reply:
I added this code to solve issue #12062.

Reply:
yes, but that shouln'd part of this PR, please extract to separate PR.

Reply:
Done.